### DISCUSSION TITLE
Has this been abandoned? Different "latest" versions everywhere.

### DISCUSSION THREAD

**Discussion by codaroma** [Huachao/vscode-restclient#1285] [Q&A]
As of 2024-07-15 there are 3 different "latest" versions
0.25.0 on github.com/Huachao/vscode-restclient/releases/tag/v0.25.0
0.25.1 on marketplace.visualstudio.com/items?itemName=humao.rest-client
0.26.0 on open-vsx.org/extension/humao/rest-client
Not clear what the differences between these 3 different latest versions are.
Anyone know what is going on?

**Comment by wilsong-mb** [comment]
Hi @codaroma , I don`t think it is abandoned; the version you see in the marketplaces is related to https://github.com/Huachao/vscode-restclient/blob/2541f7ed596b8fa944578753c94159faa839521d/package.json#L5.

I think it is just desynced, probably the author just published manually without updating the release here
